README.md

```diff
@@ -64,3 +64,105 @@ $ docker run --volume $PWD:/home/harambe/project --volume ~/.tokenlists:/home/ha
 This project is in development and should be considered a beta.
 Things might not be in their final state and breaking changes may occur.
 Comments, questions, criticisms and pull requests are welcomed.
+
+## Full Environment Execution Example
+
+Running the `Quick Usage` and `Docker Usage` examples will fail if you do not have a full environment setup.
+
+First, it is suggested that you use a virtual environment, and then install the Silverback application. Choose any of your liking. This is not a requirement, but will help with setup if you do not have a `~/.tokenslists` folder.
+
+```bash
+python3 -m venv venv
+source ./venv/bin/activate
+pip install .
+```
+
+Next, you will need a Web3 Alchemy key. If you attempt to run the `Docker Usage` command, you will get the following error:
+
+```bash
+$ docker run --volume $PWD:/home/harambe/project --volume ~/.tokenlists:/home/harambe/.tokenlists apeworx/silverback:latest run "example:app" --network :mainnet:alchemy
+Traceback (most recent call last):
+  File "/usr/local/bin/silverback", line 8, in <module>
+    sys.exit(cli())
+  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
+    return self.main(*args, **kwargs)
+  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1078, in main
+    rv = self.invoke(ctx)
+  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
+    return _process_result(sub_ctx.command.invoke(sub_ctx))
+  File "/usr/local/lib/python3.10/site-packages/ape/cli/commands.py", line 95, in invoke
+    with network_ctx as provider:
+  File "/usr/local/lib/python3.10/site-packages/ape/api/networks.py", line 679, in __enter__
+    return self.push_provider()
+  File "/usr/local/lib/python3.10/site-packages/ape/api/networks.py", line 692, in push_provider
+    self._provider.connect()
+  File "/usr/local/lib/python3.10/site-packages/ape_ethereum/provider.py", line 113, in connect_wrapper
+    connect(self)
+  File "/usr/local/lib/python3.10/site-packages/ape_alchemy/provider.py", line 102, in connect
+    self._web3 = Web3(HTTPProvider(self.uri))
+  File "/usr/local/lib/python3.10/site-packages/ape_alchemy/provider.py", line 72, in uri
+    raise MissingProjectKeyError(options)
```

Comment:
I usually gloss over unnecessary details using `...` so the reader can focus on the meaningful instructions
```suggestion
...
```